src/views/_screen-sm-max.scss

```diff
@@ -7,13 +7,13 @@
 // Heading font-sizes
 h1,
 .h1 {
-	font-size: 1.7em; // expected 34px;
-	margin-top: .75em; // expected ~15px;
+	font-size: 34px;
+	margin-top: 26px;
```

Comment:
The expectations comment for the old ``margin-top`` property was way off... in practice Gecko and Blink were computing ``.75em`` to ``25.5px``. So I rounded it up to ``26px``.